config/locales/en.yml

```diff
@@ -54,18 +54,24 @@
 #   - https://community.openfoodnetwork.org/t/workflow-to-structure-translations/932
 
 en:
+  # Overridden here due to a bug in spree i18n (Issue #870, and issue #1800)
   activerecord:
-    # Overridden here due to a bug in spree i18n (Issue #870)
     attributes:
       spree/order:
         payment_state: Payment State
         shipment_state: Shipment State
+        completed_at: Completed At
+        number: Number
+        email: Customer E-Mail
```

Comment:
where are these 3 translations used?

Reply:
In Admin/Orders list (see #1800). As you can see [here](https://github.com/spree-contrib/spree_i18n/blob/1-3-stable/config/locales/fr.yml#L53) they're not translated in spree_i18n (at least for French), it seems that adding them in `en.yml` would make them appear in Transifex and make it possible to translate them here.

Reply:
Given that we use our own fork, I'd rather fix it there https://github.com/openfoodfoundation/spree_i18n/blob/1-3-stable/config/locales/fr.yml#L53

I see it correctly translated in the `master` branch. Can you check whether it got fix in any upcoming commit? I'm afraid we might need to move a hundred commits forward as we did in the step 6 for the spree gem.

Reply:
It got fixed in a commit which is not on the 1-3-stable branch but there are commits  on the 1-3-stable branch which are not in other branches... do you think I should just PR on the 1-3-stable branch?

Reply:
> do you think I should just PR on the 1-3-stable branch?

Yes, please

Reply:
After talking with @myriamboure it seems like 2 of these translations (`number` and `email`) won't be translated the way we'd like them to be. That would be nice if I add them in OFN repo, but I can still PR for `completed_at` and `show_only_unfulfilled_orders` which will be rightly translated so don't need to be added in OFN repo.